common/src/main/java/com/wynntils/features/user/MythicBoxScalerFeature.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.user;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.wynntils.core.config.Config;
+import com.wynntils.core.features.UserFeature;
+import com.wynntils.mc.event.GroundItemEntityTransformEvent;
+import com.wynntils.wynn.item.parsers.WynnItemMatchers;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+
+public class MythicBoxScalerFeature extends UserFeature {
+
+    @Config
+    private float scale = 2f;
+
+    @SubscribeEvent
+    public void onItemRendering(GroundItemEntityTransformEvent e) {
+        if (!WynnItemMatchers.isMythicBox(e.getItemStack())) return;
+
+        PoseStack stack = e.getPoseStack();
+
+        // cancel out old scale y transform and later replace it after scaling
+        stack.translate(0f, -0.25f, 0f);
```

Comment:
This magic constant 0.25, is it depending on something Vanilla did prior to this getting called by the mixin? The comment reads a bit odd -- it says cancel "scale y" but it does this by doing a translate. Don't you mean that you cancel out the translation? Also, maybe make a final static constant of the magic number?

Reply:
>I think you should extract this magic value to a constant, and make a good name or a comment about what kind of value this is, how you did find it (in case it changes in Vanilla and needs updating etc).

I will do the later, but don't think there is any point in extracting when it only appears twice in quick succession

> Effectively, vanilla code plays out as
```java
        stack.translate(0f, -0.25f * model_scale, 0f); // transform y of item on ground
        ...
        stack.scale(model_scale, model_scale, model_scale); // scale model
```

We inject after the `scale` application so that it becomes

```java
        stack.translate(0f, -0.25f * model_scale, 0f); // transform y of item on ground
        ...
        stack.scale(model_scale, model_scale, model_scale); // scale model
        stack.translate(0f, -0.25f, 0f); // new code
        stack.scale(scale, scale, scale); // new code
        stack.translate(0f, -0.25f, 0f); // new code
```

which should be equivalent to
```java
        stack.translate(0f, -0.25f * model_scale * scale, 0f); // transform y of item on ground
        ...
        stack.scale(model_scale * scale, model_scale * scale, model_scale * scale); // scale model
```

Reply:
I think this was an excellent explanation! Can you please add this, or a revised version of this, as a comment to this code?